src/content/operations/usage/_index.en.md

```diff
@@ -244,17 +244,44 @@ $ kubectl config use-context cluster3
 Switched to context "cluster3".
 ```
 
-The following commands create an `nginx` Service in the nginx-test namespace which targets TCP port 8080 on any Pod with the `app: nginx`
-label and exposes it on an abstracted Service port. When created, the Service is assigned a unique IP address (also called ClusterIP):
+The following commands create an `nginx` Service in the nginx-test namespace which targets TCP port 8080, with name http, on any Pod with the
+`app: nginx` label and exposes it on an abstracted Service port. When created, the Service is assigned a unique IP address
+(also called ClusterIP):
```

Comment:
```suggestion
(also called ClusterIP):
```

Is this an API object? If so, it should be `code style`.

https://kubernetes.io/docs/contribute/style/style-guide/#code-style-inline-code